GotExplorer.API/Configuration/CorsSettings.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Cors.Infrastructure;
+
+namespace GotExplorer.API.Configuration
+{
+    public class CorsSettings
+    {
+        public string[]? Origins { get; set; }
+        public string[]? Headers { get; set; }
+        public string[]? Methods { get; set; }
+
+        public CorsPolicy GetPolicy()
+        {
+            CorsPolicyBuilder builder = new CorsPolicyBuilder();
+            if (Headers == null || Headers.Length == 0)
+                builder.AllowAnyHeader();
+            else
+                builder.WithHeaders(Headers);
+
+
+            if (Methods == null || Methods.Length == 0)
+                builder.AllowAnyMethod();
+            else
+                builder.WithMethods(Methods);
+
+
+            if (Origins == null || Origins.Length == 0)
+                builder.AllowAnyOrigin();
+            else
+                builder.WithHeaders(Origins);
+            
+            return builder.Build();
```

Comment:
I suggest that this class should not exist. I would recommend to avoid use AllowAnyOrigin, AllowAnyMethod, AllowAnyHeader at total